gc/base/MemorySpace.hpp

```diff
@@ -185,8 +185,8 @@ class MM_MemorySpace : public MM_BaseVirtual
 
 	static MM_MemorySpace *getMemorySpace(void *memorySpace) { return (MM_MemorySpace *)memorySpace; }
 
-	uintptr_t releaseFreeMemoryPages(MM_EnvironmentBase* env);
-	
+	virtual uintptr_t releaseFreeMemoryPages(MM_EnvironmentBase* env, uintptr_t memoryType);
```

Comment:
so far, we had no subclasses of MemorySpace, so I'd rather not have virtual (although I'll admit there are already a couple of methods in this class that use virtual, but also for no good reason)